ci-scripts/dockerfiles/Lmod.dockerfile

```diff
@@ -0,0 +1,41 @@
+#
+# Execute this from the top-level ReFrame source directory
+#
+
+FROM ubuntu:20.04
+
+ENV TZ=Europe/Zurich
+ENV DEBIAN_FRONTEND=noninteractive
+ENV _LMOD_VER=8.4.12
+WORKDIR /root
+
+# ReFrame requirements
+RUN \
+  apt-get -y update && \
+  apt-get -y install gcc && \
+  apt-get -y install make && \
+  apt-get -y install git && \
+  apt-get -y install python3
```

Comment:
```suggestion
  apt-get -y install python3 python3-pip
```

Reply:
Why is this needed? I guess for the `./bootstrap.sh`, but why did it work for me? Because of the `COPY`?

Reply:
I guess it has copied also the external that you had.